ykdl/util/wrap.py

```diff
@@ -34,13 +34,13 @@ def launch_player(player, urls, ext, **args):
         if ext == 'm3u8' and any(os.path.isfile(url) for url in urls):
             cmd += ['--demuxer-lavf-o', 'protocol_whitelist=[file,http,https,tls,rtp,tcp,udp,crypto,httpproxy]']
```

Comment:
Also this one is.

Reply:
I've updated this, but I don't have a source to test on it. Hopefully it works fine. : )